### PR TITLE
Added comments and improved structure

### LINKED ISSUE #48: Code Cleanup - Add Comments and Improve Variable Names
In this instance of cleanup one needs to add comments where code gets ambiguous. Additionally, make the variable names more intuitive. The aim of this is to make the code easier to understand overall.

### PR DESCRIPTION
**Pull request by bb511**
#### Description of Work

Fixes #48 
As the title says, added comments, improved variable names and improved structure.
Note: Code still needs to be organised in modules, right now it is too messy and still kind of hard to understand.

#### Testing Instructions

1. Run `example_runScripts.py`  in the mantidpython console.
2. Check if the tables work properly.

### CODE REVIEW

**Comment by Anders-Markvardsen** [comment]
merge conflicts

File: fitbenchmarking/plotHelper.py [hunk 1]
```diff
@@ -64,48 +65,25 @@ def order_data(self):
...
-        @param save:: the name of the file to save to
-        the default is not to save
-        """
-        self.data.append(inputData)
-
-
-class plot(data,insert):
+class plot(data):
     """
     Minimal information for a plot
```
Comment (Anders-Markvardsen) [inline fitbenchmarking/plotHelper.py]: Would you be able to expand on "Minimal information for a plot" documentation please

**Comment by bb511** [comment]
None means that it has no predefined parameters.

**Comment by Anders-Markvardsen** [comment]
OK as discussed change 'None' to 'Use default parameter value' and move table up to section above

**Comment by Anders-Markvardsen** [comment]
I meant Use default parameter values

**Comment by Anders-Markvardsen** [comment]
tested and works

**Pull request merged**